content/id/docs/concepts/services-networking/dual-stack.md

```diff
@@ -0,0 +1,145 @@
+
+{{% capture body %}}
+
+## Fitur-fitur yang didukung
+
+Mengaktifkan _dual-stack_ IPv4 / IPv6 pada klaster Kubernetes kamu untuk 
+menyediakan fitur-fitur berikut ini:
+
+   * Jaringan Pod _dual-stack_ (pengalokasian sebuah alamat IPv4 dan IPv6 
+     untuk setiap Pod)
+   * Layanan yang mendukung IPv4 dan IPv6 (setiap layanan hanya untuk satu 
+     keluarga alamat)
+   * Pod routing untuk keluar klaster (misalnya Internet) melalui antarmuka 
+     IPv4 dan IPv6
+
+## Prasyarat
+
+Prasyarat berikut diperlukan untuk menggunakan _dual-stack_ IPv4/IPv6 pada 
+klaster Kubernetes :
+
+   * Kubernetes versi 1.16 atau yang lebih baru
+   * Dukungan dari penyedia layanan akan jaringan _dual-stack_ (Penyedia layanan
+     _cloud_ atau yang lainnya harus dapat memberikan Node Kubernetes dengan 
+     antarmuka jaringan IPv4/IPv6 yang dapat dirutekan)
+   * Sebuah _plugin_ jaringan yang mendukung _dual-stack_ (seperti Kubenet atau Calico)
+   * Kube-proxy berjalan dalam mode IPVS
+
+## Mengaktifkan _dual-stack_ IPv4/IPv6
+
+Untuk mengaktifkan _dual-stack_ IPv4/IPv6, aktifkan `IPv6DualStack` [gerbang fitur](../en/docs/reference/command-line-tools-reference/feature-gates/) 
+untuk komponen yang relevan dari klaster kamu, dan atur pengalokasian jaringan 
+_dual-stack_ pada klaster:
+
+   * kube-controller-manager:
+      * `--feature-gates="IPv6DualStack=true"`
+      * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>` misalnya `--cluster-cidr=10.244.0.0/16,fc00::/24`
+      * `--service-cluster-ip-range=<IPv4 CIDR>,<IPv6 CIDR>`
+      * `--node-cidr-mask-size-ipv4|--node-cidr-mask-size-ipv6` _default_ /24 
+        untuk IPv4 dan /64 untuk IPv6
+   * kubelet:
+      * `--feature-gates="IPv6DualStack=true"`
+   * kube-proxy:
+      * `--proxy-mode=ipvs`
+      * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>`
+      * `--feature-gates="IPv6DualStack=true"`
+
+{{< caution >}}
+Jika kamu menentukan blok alamat IPv6 yang lebih besar dari /24 melalui 
+`--cluster-cidr` pada baris perintah, maka pengalokasian tersebut akan gagal.
+{{< /caution >}}
+
+## Layanan-layanan
+
+Jika klaster kamu mengaktifkan jaringan _dual-stack_ IPv4/IPv6, maka kamu dapat 
+membuat {{<glossary_tooltip text ="Service" term_id ="service">}} (layanan) dengan 
+alamat IPv4 atau IPv6. Kamu dapat memilih keluarga alamat untuk IP klaster  
+layanan kamu dengan mengatur bagian, `.spec.ipFamily`, pada layanan tersebut.
+Kamu hanya dapat mengatur bagian ini saat membuat layanan baru. Mengatur bagian 
+`.spec.ipFamily` bersifat opsional dan hanya boleh digunakan jika kamu berencana
+untuk mengaktifkan {{<glossary_tooltip text ="Service "term_id ="service">}} 
+IPv4 dan IPv6 dan {{<glossary_tooltip text ="Ingress "term_id ="ingress">}}
+pada klaster kamu. Konfigurasi bagian ini bukan persyaratan untuk lalu lintas 
+[_egress_] (#trafik-egress).
+
+{{< note >}}
+Keluarga alamat _default_ untuk klaster kamu adalah keluarga alamat dari rentang 
+IP cluster layanan pertama yang dikonfigurasi melalui flag 
```

Comment:
```suggestion
IP cluster Service pertama yang dikonfigurasi melalui parameter 
```
wdyt if we use _parameter_ instead of _flag_?

Reply:
Parameter usually also refers to "command line argument" which usually refers to the positional argument.
Flags/options in CLI context refer to e.g. `--xxx-yyy` and `-x`. We might need to keep it as "flag" and italicize it.

Reply:
nice catch, what about `opsi`?
currently discussed on https://github.com/kubernetes/website/pull/20166

Reply:
`opsi` is better.